application/orchestrator/tests/test_orchestrator.py

```diff
@@ -4,13 +4,12 @@
 from unittest.mock import MagicMock, patch
 
 from aws_lambda_powertools.utilities.typing.lambda_context import LambdaContext
+from orchestrator.orchestrator import invoke_lambda, lambda_handler
 from pytest import approx, fixture
 
-from application.orchestrator.orchestrator import invoke_lambda, lambda_handler
-
 from common.types import UpdateRequestMetadata, UpdateRequestQueueItem
 
-FILE_PATH = "application.orchestrator.orchestrator"
+FILE_PATH = "orchestrator.orchestrator"
```

Comment:
Is there a reason this has changed? I believe it stops sonarcloud from properly tracking coverage